### PR TITLE
Added prefix for header with jwt token

### PR DESCRIPTION
**Pull request by artemlisikh**
Removed redundant field in JwtAuthController

### CODE REVIEW

**Comment by artemlisikh** [comment]
Hey, @alisiikh, whould you mind taking a look?

File: src/main/java/com/geronimo/config/security/jwt/filter/JwtAuthTokenFilter.java [hunk 1]
```diff
@@ -32,9 +32,6 @@
     @Value("${jwt.header}")
     private String tokenHeaderName;
 
-    @Value("${jwt.scheme}")
-    private String scheme;
```
Comment (olisikh) [inline src/main/java/com/geronimo/config/security/jwt/filter/JwtAuthTokenFilter.java:36]: Ok, I see it wasn't used in this filter, right?

Comment (artemlisikh) [reply src/main/java/com/geronimo/config/security/jwt/filter/JwtAuthTokenFilter.java:36]: exactly

Comment (olisikh) [reply src/main/java/com/geronimo/config/security/jwt/filter/JwtAuthTokenFilter.java:36]: I moved that stuff to JwtUtil, also removed this field from JwtAuthController :)

Comment (artemlisikh) [reply src/main/java/com/geronimo/config/security/jwt/filter/JwtAuthTokenFilter.java:36]: cool

**Review by olisikh — APPROVED** [review]
Good!

**Pull request merged**